Conductor/Client/Configuration.cs

```diff
@@ -134,6 +135,29 @@ public Configuration()
             }
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Configuration" /> class
+        /// </summary>
+        public Configuration(
+            IDictionary<string, string> defaultHeaders,
+            string keyId,
```

Comment:
We should assign this to keyId and keySecret and create AuthenticationSettings from this.